test/extended/imageapis/limitrange_admission.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const limitRangeName = "limits"
 
-var _ = g.Describe("[Feature:ImageQuota][registry][Serial] Image limit range", func() {
+var _ = g.Describe("[Feature:ImageQuota][registry][Serial][Suite:openshift/registry/serial] Image limit range", func() {
```

Comment:
At the end, please

Reply:
ginkgo concatenates this text w/ the additional test text, so i think it makes more sense to put the tags at the beginning so you don't get a weird concatenation with the tag in the middle.  What's the argument for putting it at the end?

Reply:
that's where the automatic suite goes

Reply:
well that seems like a bad idea too.  why double down on it? :)